x/currency/errors.go

```diff
@@ -0,0 +1,39 @@
+package currency
+
+import (
+	stderrors "errors"
+	fmt "fmt"
+
+	"github.com/iov-one/weave/errors"
+)
+
+const (
+	CodeInvalidToken = 2000
+
+	CodeInvalidObject = 1100 // TODO: this is the same as in x/namecoin
```

Comment:
This has to have a different range now for consistency.
You have to document which range of error code the module reserves similar to other modules.

Reply:
namecoin has this code declared with a TODO to extract it to top level package. This is exactly the code that I need, so I have reused it. I would love to redo the errors, because searching for a free error code range is a challenge anyway.

Reply:
@husio I think it does make sense to use a different range/numbers. Or you can split namecoin range into 2, extract what is needed to your new module and leave it like this. It is important that there are no clashes and also that it is easy to add a couple of new errors just in case.

Reply:
ja, either move this to a top level package or new range